cli/cmd/root.go

```diff
@@ -64,6 +64,6 @@ func setLogLevel(logLevel string) {
 	case "panic":
 		logrus.SetLevel(logrus.PanicLevel)
 	default:
-		logrus.Fatal("Unrecognized log level entry: %s", logLevel)
```

Comment:
This looks like a typo, should be FatalF() I believe.

Reply:
logrus.Fatalf( .... )

Reply:
thanka